backend/.eslintrc

```diff
@@ -32,13 +30,16 @@
     "process": "readonly",
     "document": "readonly"
   },
-  "settings": {
-  },
+  "settings": {},
   "rules": {
-    "no-restricted-properties": [ "error", {
-      "property": "toString",
-      "message": "e.toString() should be fastify.log.error(e, 'your string'). Other use-cases should avoid obj.toString() on principle. Craft the string you want instead."
-    }],
+    "eqeqeq": ["error", "always"],
```

Comment:
These are some other options we can use: https://eslint.org/docs/latest/rules/eqeqeq#options

Reply:
@manaswinidas please include option `{ null: 'ignore' }`. We shouldn't have to distinguish between `undefined` and `null`.